charts/lighthouse/.helmignore

```diff
@@ -19,3 +19,6 @@
 .project
 .idea/
 *.tmproj
+# Project files
+Makefile
```

Comment:
Is the Makefile even used?

Reply:
I don't know but it should not end up in the package.